CHANGELOG.rst

```diff
@@ -21,3 +21,51 @@ Added
 ~~~~~
 
 * First release on PyPI.
+
+[2.12.3] - 2020-05-06
+---------------------
+
+Added
+~~~~~
+
+* Improve the way that we can filter sentry exceptions.
+
+[2.12.1] - 2020-04-16
+---------------------
+
+Added
+~~~~~
+
+* Added a completely configurable OpenId Connect based backend for third party auth.
+
+[2.11.1] - 2020-04-15
+---------------------
+
+Added
+~~~~~
+
+* Use USERNAME_MAX_LENGTH defined in edx-platform.
+
+[2.9.0] - 2020-04-06
+--------------------
+
+Added
+~~~~~
+
+* Add capability to ignore exceptions in sentry.
+
+[2.8.0] - 2020-03-20
+--------------------
+
+Added
+~~~~~
+
+* Adding sentry integration
+
+[2.6.0] - 2020-01-09
+--------------------
+
+Removed
+~~~~~~~
+
+* Remove microsite configuration mentions.
```

Comment:
@morenol  Could you please add this release:
0.14.0
Added
Course management automation.
Description
This new Studio module allows you to make changes to the course configuration for several courses at once.
More information: https://github.com/eduNEXT/eox-core#course-management-automation
Added
Linting tests (pylint - eslint) to CircleCI test suites.

Reply:
done

Reply:
Thanks.